### DISCUSSION TITLE
part-ii/2019-118/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#206] [Question comments]
Part II 2019 Probability and Measure: Paper 3, Section II, K 

https://questions.tripos.org/part-ii/2019-118/

**Comment by eha1234** [comment]
<details><summary>(c)</summary>

The moment generating functions have to agree (or the characteristic function I guess)
</details>

<details><summary>(d)</summary>

Set $X^1 = e^{-Z}$
</details>
